trx/fetcher.py

```diff
@@ -50,13 +61,21 @@ def fetch_data(files_dict, keys=None):
         keys = files_dict.keys()
     elif isinstance(keys, str):
         keys = [keys]
+
     for f in keys:
-        url, md5 = files_dict[f]
+        url, expected_md5 = files_dict[f]
         full_path = os.path.join(trx_home, f)
 
         logging.info('Downloading {} to {}'.format(f, trx_home))
-        cached_download(url=GOOGLE_URL+url,
-                        path=full_path,
-                        md5=md5,
-                        quiet=True,
-                        postprocess=extractall)
+        if not os.path.exists(full_path):
+            urllib.request.urlretrieve(url, full_path)
+
+        actual_md5 = md5sum(full_path)
+        if expected_md5 != actual_md5:
+            raise ValueError(f'md5sum for {f} does not match.')
+
+        if f.endswith('.zip'):
+            dst_dir = os.path.join(trx_home, f[:-4])
+            shutil.unpack_archive(full_path,
+                                  extract_dir=dst_dir,
```

Comment:
Instead of `dst_dir` here use `trx_home` (the zip contains the DSI/ directory itself)
OR
Remove one layer in the zip file (and recompute the MD5SUM)

I don't know if one is better than the other (but the first is simpler)